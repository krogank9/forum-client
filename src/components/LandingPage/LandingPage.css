.forum-logo {
  margin: auto;
  height: 135px;
  width: auto;
  margin-top: 50px;
}