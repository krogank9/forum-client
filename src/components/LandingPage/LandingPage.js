import React, { Component } from 'react';
import './LandingPage.css';

//import { Link } from "react-router-dom";

class LandingPage extends Component {
  render() {
    return (
      <div>
        <img className="nav-logo" src={`${process.env.PUBLIC_URL}/assets/logo.svg`} alt="Logan Krumbhaar's Forum" className="forum-logo"></img>
        <h1>Welcome to my forum</h1>
        <p>Click the "View boards" button above to start browsing the various categories and topics.</p>
      </div>
    );
  }
}

export default LandingPage;