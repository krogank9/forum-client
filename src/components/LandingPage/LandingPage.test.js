import React from 'react';
import ReactDOM from 'react-dom';
import LandingPage from './LandingPage';

import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`LandingPage component`, () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<LandingPage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });


  it('renders the LandingPage', () => {
    const wrapper = shallow(
      <LandingPage />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
