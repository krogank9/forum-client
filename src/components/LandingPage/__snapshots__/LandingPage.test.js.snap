// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders the LandingPage 1`] = `
<div
  className="landing-page-description"
>
  <img
    alt="Logan Krumbhaar's Forum"
    className="forum-logo"
    src="/assets/logo.svg"
  />
  <h1>
    Welcome to my forum
  </h1>
  <p>
    Click the "View boards" button in the navigation bar above to start browsing the various categories and topics.
  </p>
  <p>
    To login to a demo account, use the username "test" with the password "abc123".
  </p>
  <p>
    <a
      href="https://github.com/krogank9/forum-client"
    >
      Client
    </a>
    : React, HTML, CSS. 
    <a
      href="https://github.com/krogank9/forum-server"
    >
      Server
    </a>
    : Node, ExpressJS, PostgreSQL.
  </p>
  <p>
    Created by 
    <a
      href="http://ltkdigital.com"
    >
      Logan Krumbhaar
    </a>
    .
  </p>
</div>
`;
